release-notes/release-notes-2-2.md

```diff
@@ -1,38 +1,22 @@
 <!-- SPDX-License-Identifier: CC-BY-4.0 -->
 <!-- Copyright Contributors to the ODPi Egeria project. -->
 
-# Release 2.2 (Planned August 2020)
+# Release 2.2 (August 2020)
 
-Release 2.2 focuses on providing the secured self-service data access point
-for a data lake.  There are two main features:
+Below are the highlights of this release:
 
-  * **Metadata Controlled Security** where security tags based on the metadata attachments to the assets
-    drive the enforcement engine managing access control
-  * **Data Virtualization Configuration** where the data virtualization engine that is providing
-    the endpoint for the self-service access point.
+ * Additional connectors are now placed in our assembly under server/lib without dependencies. If you need to use a connector that requires additional dependencies that are not alreadypart of the server chassis, you will need to add those libraries here.
+ * The docker image has been updated to use a later openjdk alpine base image due to incompatabilities with our Janus support in the old images for java 8.
```

Comment:
Is there only one docker image? It might be worth having this as a link to it.

Reply:
added